app/services/code_climate/update_project_service.rb

```diff
@@ -0,0 +1,47 @@
+module CodeClimate
+  class UpdateProjectService < BaseService
+    CODE_CLIMATE_API_ORG_NAME = 'rootstrap'.freeze
+
+    def initialize(project)
+      @project = project
+    end
+
+    def call
+      return unless update_metric? && code_climate_project_info
+
+      create_project_code_climate_metric unless project_code_climate_metric
+
+      project_code_climate_metric.update!(
+        code_climate_rate: code_climate_project_info.rate,
+        invalid_issues_count: code_climate_project_info.invalid_issues_count,
+        wont_fix_issues_count: code_climate_project_info.wont_fix_issues_count
```

Comment:
Can we also include the number of open issues? Sorry, I forgot to add it to the wireframe 😅

Reply:
Yes, it should be possible.
I have a question though.
CodeClimate issue status are `[new, open, confirmed, invalid, wontfix]`. I don't know what's the difference between `new` and `open`. Would `#new + #count + #confirmed`be  the number you are looking for?

Reply:
Done the backend task.
I've added a card to the backlog with the task for showing it in the frontend.